go.mod

```
@@ -1,6 +1,6 @@
 module github.com/spacemeshos/go-spacemesh
 
-go 1.23.4
+go 1.24.0
```

There's already a 1.24.1 with a CVE fix :)
```suggestion
go 1.24.1
```